packages/api/models/granules.js

```diff
@@ -255,12 +282,22 @@ class Granule extends Manager {
       published: g.published
     });
 
-    return this.update(
+    const updateResponse = await this.update(
       { granuleId: g.granuleId },
       {
         files: updatedFiles.map(partial(renameProperty, 'name', 'fileName'))
       }
     );
+
+    if (!process.env.noSNS) {
+      const snsRecord = {
+        event: 'Update',
+        record: updateResponse
+      };
+      await publishGranuleSnsMessage(snsRecord);
```

Comment:
This can also be put to `this.update` method rather than individual methods (CMR action, move granule).
Does the record has full granule information?

Reply:
No, this will look like
updateResponse: {
  "granuleId": "MOD11A1.A2017137.h19v16.006.2017138085750",
  "action": "move",
  "status": "SUCCESS"
}

Reply:
Metrics needs full granule record, it won't know what to do with this updates.